src/mpvis/mpdfg/actions.py

```diff
@@ -199,6 +202,7 @@ def view_multi_perspective_dfg(
     cost_currency: str = "USD",
     rankdir: str = "TD",
     format: str = "svg",
+    arc_thickness_by: str = "frequency",
```

Comment:
[REQ]
Use string literals.

```suggestion
    arc_thickness_by: Literal["frequency", "time"] = "frequency",
```